### DISCUSSION TITLE
Updating the Maximum Event Body Size

### DISCUSSION THREAD

**Discussion by mitchellsuzuki** [datalust/seq-tickets#2273] [Help]
Hi SEQ,
I am using FluentBit to send all logs from my kubernetes cluster in to SEQ using the HTTP ingestion endpoint. `/ingest/clef`
For the most part, it's working great. However I have a number of events that are failing to be ingested due to exceeding the maximum event body size.

How do I update this value?
For reference, I am deploying SEQ into my k8s cluster using the datalust/seq helm chart and am running the latest version 2024.3.1

Alternatively, I'd like to enable compression, however it would appear that SEQ does not currently support this.

**Comment by liammclennan** [comment]
Hi @mitchellsuzuki 

Go to Settings > System and there are settings for 'Raw ingestion payload limit' and 'Raw event body limit'.

**Reply by jmsuzuki** [reply]
if I wanted to do this via api or helm installation config. how would I do this?
one of my team's requirements is "no manual steps in production".

**Reply by liammclennan** [reply]
@jmsuzuki there is the [seqcli](https://docs.datalust.co/docs/command-line-client#setting-set), or the [HTTP API](https://docs.datalust.co/docs/server-http-api#apisettings).

**Reply by jmsuzuki** [reply]
Hi Liam, from the HTTP API docs, I can't find the settings I'm looking for.

From another discussion though on the topic, I do see that you suggested using this endpoint:
`/api/settings/setting-raweventmaximumcontentlength`

I assume this is a `PUT` request like the other settings are?

Can you provide an example body for the request?

**Reply by liammclennan** [reply]
Yes, it is a `PUT`. Here is an example body:

```json
{
  "Name":"raweventmaximumcontentlength",
  "Value":999999,
  "Id":"setting-raweventmaximumcontentlength",
  "Links":{"Self":"api/settings/setting-raweventmaximumcontentlength","Group":"api/settings/resources"}
}
```

I got this from the browser developer tools by saving the settings. 

I'm not sure why it's not in the HTTP API docs - we'll get that fixed.
